src/components/input/AmountField.tsx

```diff
@@ -4,22 +4,29 @@ import { OutlineButton } from 'src/components/buttons/OutlineButton';
 import { NumberField } from 'src/components/input/NumberField';
 import { formatNumberString } from 'src/components/numbers/Amount';
 import { MIN_REMAINING_BALANCE } from 'src/config/consts';
+import { TokenId } from 'src/config/tokens';
 import { fromWei } from 'src/utils/amount';
 
 export function AmountField({
   maxValueWei,
   maxDescription,
   disabled,
+  tokenId,
 }: {
   maxValueWei: bigint;
   maxDescription: string;
   disabled?: boolean;
+  tokenId: TokenId;
 }) {
   const { setFieldValue } = useFormikContext();
 
   const maxValue = useMemo(
-    () => Math.max(0, fromWei(maxValueWei - MIN_REMAINING_BALANCE)),
-    [maxValueWei],
+    () =>
+      Math.max(
+        0,
+        fromWei(tokenId === TokenId.CELO ? maxValueWei - MIN_REMAINING_BALANCE : maxValueWei),
```

Comment:
only subtract MIN amount when token is CELO! otherwise cant withdraw all st celo or locked celo